ants/registration/build_template.py

```diff
@@ -112,6 +135,7 @@ def build_template(
             wavgfn = mktemp(suffix=".nii.gz")
```

Comment:
`afffn` and `wavgfn` should be written to `work_dir`, i.e. so they can be used later or get deleted with the rest.